mimic/rest/maas_api.py

```diff
@@ -1158,6 +1158,35 @@ def view_agent_host_info(self, request, tenant_id):
                          'next_marker': None,
                          'next_href': None}})
 
+    @app.route('/v1.0/<string:tenant_id>/views/connections', methods=['GET'])
+    def view_connections(self, request, tenant_id):
+        """
+        Lists agent connections.
```

Comment:
I don't have one yet as it's a new API call. It's similar to https://developer.rackspace.com/docs/cloud-monitoring/v1/developer-guide/#list-agent-connections , but allows you to list connections for many agents at once.